### PR TITLE
Fix: Load lightbox styles with breakpoints [ED-15794]

### PR DESCRIPTION
**Pull request by murzin-nikita**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: tests/playwright/sanity/basic-gallery.test.ts [hunk 1]
```diff
@@ -60,6 +62,34 @@ test( 'Basic Gallery Lightbox test with older Swiper', async ( { page, apiReques
...
+	await editor.openPanelTab( 'content' );
+	await editor.addImagesToGalleryControl();
+	await editor.setSelectControlValue( 'open_lightbox', 'yes' );
+
+	await editor.publishPage();
+
+	await breakpoints.setBreakpoint( editor, 'mobile', viewportSize.mobile.width );
+
+	// Act.
+	const pageId = await editor.getPageId();
```
Comment (hein-obox) [inline tests/playwright/sanity/basic-gallery.test.ts:83]: Maybe extract to `viewPage()`, as we have in the Pro repo: 

https://github.com/elementor/elementor-pro/blob/a526f2337aa09480b47a75f37a87493750f6c437/tests/playwright/pages/editor-page.ts#L501

File: tests/playwright/sanity/basic-gallery.test.ts [hunk 2]
```diff
@@ -60,6 +62,34 @@ test( 'Basic Gallery Lightbox test with older Swiper', async ( { page, apiReques
...
+	await page.goto( `/?p=${ pageId }` );
+	await page.setViewportSize( viewportSize.mobile );
+	await page.locator( 'div#gallery-1 img' ).first().click();
+	await editor.page.waitForTimeout( 1000 );
+
+	// Assert.
```
Comment (hein-obox) [inline tests/playwright/sanity/basic-gallery.test.ts:89]: For clarity: is the padding of `70px 0` applied within this screenshot? Would it fail, if the breakpoint wouldn't work?

File: tests/playwright/sanity/basic-gallery.test.ts [hunk 3]
```diff
@@ -60,6 +62,34 @@ test( 'Basic Gallery Lightbox test with older Swiper', async ( { page, apiReques
...
+	await editor.setSelectControlValue( 'open_lightbox', 'yes' );
+
+	await editor.publishPage();
+
+	await breakpoints.setBreakpoint( editor, 'mobile', viewportSize.mobile.width );
+
+	// Act.
+	const pageId = await editor.getPageId();
+	await page.goto( `/?p=${ pageId }` );
+	await page.setViewportSize( viewportSize.mobile );
```
Comment (hein-obox) [inline tests/playwright/sanity/basic-gallery.test.ts:85]: I am thinking the following: 

1. If the mobile breakpoint is 400px, then we should test a screensize in between 400px and 767px, to ensure that the `padding: 70px` is applied. We could test a breakpoint of 500px.
2. Additionally we could test the breakpoint of 350px, to verify that the `padding: 70px 0` is applied.

Screenshot testing is sufficient, but we need to use a correct breakpoint.

File: assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
-@import "../../helpers/variables";
-@import "../breakpoints/breakpoints";
```
Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss:2]: @mykytamurzin Can you please apply this to the following file as well?

`assets/dev/scss/frontend/conditionals/with-breakpoints/apple-webkit.scss`

**Review by hein-obox — APPROVED** [review]

**Review by hein-obox — APPROVED** [review]

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**